src/main/java/lotto/step2/domain/Lottos.java

```diff
@@ -0,0 +1,28 @@
+package lotto.step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Lottos {
+    private List<Lotto> lottos;
+
+    public Lottos(LottoStrategy lottoStrategy, LottoCount lottCount) {
+        this.lottos = createLottos(lottoStrategy, lottCount);
+    }
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    private List<Lotto> createLottos(LottoStrategy lottoStrategy, LottoCount lottCount) {
+        List<Lotto> lottoList = new ArrayList<>();
+        IntStream.range(0, lottCount.getLottoCount())
+                .forEach(i -> lottoList.add(new Lotto(lottoStrategy.createLotto().getLotto())));
+        return lottoList;
+    }
```

Comment:
```suggestion
    private List<Lotto> createLottos(LottoStrategy lottoStrategy, LottoCount lottCount) {
        return lottCount.play(lottoStrategy);
    }
```
```java
public class LottoCount {
...
    public List<Lotto> play(LottoStrategy lottoStrategy) {
        return IntStream.range(0, lottoCount)
                .mapToObj(i -> lottoStrategy.createLotto())
                .collect(Collectors.toList());
    }
```

- 위와 같이 getter를 사용하기 보다 각 객체에 메시지를 보내는 것에 대해서는 어떻게 생각하시나요?

Reply:
훨씬 깔끔하네요...!
객체에 직접 필요한걸 요청하는게 좋네요!
앞으로 메시지 보내는 부분 적용 많이 해보겠습니다~
감사합니다!